R/system.R

```diff
@@ -793,14 +793,14 @@ getListOfColumns <- function(type, host, port, databaseName, username, password,
 
 #' API to execute a query that can be handled with DBI
 #' @export
-executeGenericQuery <- function(type, host, port, databaseName, username, password, query, catalog = "", schema = ""){
+executeGenericQuery <- function(type, host, port, databaseName, username, password, query, catalog = "", schema = "", n = -1){
```

Comment:
Other similar function in this file uses "numOfRows" as the name for this parameter. Let't make it consistent.

Reply:
Changed argument name to `numOfRows`